AUR/PKGBUILD

```diff
@@ -2,39 +2,38 @@
 # Maintainer: Antoine Bertin <ant.bertin@gmail.com>
 # Maintainer: Andrey Kolchenko <andrey@kolchenko.me>
 
-pkgname=linux-enable-ir-emitter
-pkgver=20210725.0
+pkgname=linux-enable-ir-emitter-git
+pkgver=r167.04f6c73
 pkgrel=1
 pkgdesc='Enables infrared cameras that are not directly enabled out-of-the box.'
+arch=(x86_64)
```

Comment:
Can you please quote all values?

Reply:
Quoting in arch=() license=() depends=(), etc is effectively pointless when not needed

Reply:
If for you quoting everything make the pkgbuild more readable, to me have the opposite effect